pkg/skaffold/kubernetes/manifest/errors.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package manifest
+
+import (
+	"fmt"
+
+	sErrors "github.com/GoogleContainerTools/skaffold/pkg/skaffold/errors"
+	"github.com/GoogleContainerTools/skaffold/proto"
+)
+
+func replaceImageErr(err error) error {
+	return sErrors.NewError(
+		proto.ActionableErr{
+			Message: fmt.Sprintf("replacing images in manifest: %s", err),
+			ErrCode: proto.StatusCode_DEPLOY_REPLACE_IMAGE_ERR,
+		})
+}
+
+func transformManifestErr(err error) error {
+	return sErrors.NewError(
+		proto.ActionableErr{
+			Message: fmt.Sprintf("unable to transform manifests: %s", err),
+			ErrCode: proto.StatusCode_DEPLOY_TRANSFORM_MANIFEST_ERR,
+		})
+}
+
+func setLabelErr(err error) error {
+	return sErrors.NewError(
+		proto.ActionableErr{
+			Message: fmt.Sprintf("setting labels in manifests: %s", err),
+			ErrCode: proto.StatusCode_DEPLOY_SET_LABEL_ERR,
+		})
+}
+
+func getImagesErr(err error) error {
```

Comment:
`imageGettingErr` or `imageRetrieveErr`?

(the `DEPLOY_GET_IMAGE_ERR` error code is singular)

Reply:
A Manigest can have one or more images. I Changed the error code to DEPLOY_GET_IMAGES_ERR